unit_test/sparse/Test_Sparse_spmv_blockcrs.hpp

```diff
@@ -156,8 +157,8 @@ void check_blockcrs_times_v(const char fOp[], scalar_t alpha, scalar_t beta,
     }  // for (lno_t ir = 0; ir < mat_b1.numRows(); ++ir)
 
     // Create the CrsMatrix for the reference computation
-    crsMat_t Acrs("new_crs_matr", nRow, nCol, nnz, &mat_val[0], &mat_rowmap[0],
-                  &mat_colidx[0]);
+    crsMat_t Acrs("new_crs_matr", nRow, nCol, nnz, mat_val, mat_rowmap,
```

Comment:
I believe you want to do something like: `mat_val.data()`?

Reply:
@lubcv 
I have a question about CrsMatrix constructors.
I agree with you that `mat_rowmap` stores `size_type` (as opposed to `ordinal_type`).
The constructor 
https://github.com/kokkos/kokkos-kernels/blob/e4529f966d2bad5c4cdaec6911bef44a8013d5fb/src/sparse/KokkosSparse_CrsMatrix.hpp#L618
expects `OrdinalType*` for rowmap.
I was expected `size_type*`. No?

Reply:
Yep, that looks like a bug to me, the constructor should use `size_type* row_map`

Reply:
Should I create an issue?

Reply:
Yes, please create an issue and I will create a PR that fixes this, hopefully it will be fairly quick to merge it.

Reply:
Created issue #1316

Reply:
Actually if you use `Kokkos::View` instead of `std::vector` the code will work just fine.
You would need to replace
```c++
std::vector<size_type> mat_rowmap(nRow+1, 0);
std::vector<lno_t> mat_colidx(nnz, 0);
```
with:
```c++
typename crsMat_t::row_map_type::non_const_type mat_rowmap(...);
typename crsMat_t::index_type::non_const_type mat_colidx(...);
typename crsMat_t::values_type::non_const_type mat_val(...);
```
You might need to create mirror view to fill these on host and deep_copy them back to the original views, afterwards the constructor will work based on the views.